docs/source/features/azureml_integration.md

```diff
@@ -0,0 +1,104 @@
+# Azure ML integration
+
+This documents outlines the integrations between Olive and Azure Machine Learning. Discover how to optimize your Azure Machine Learning assets within Olive.
+
```

Comment:
.... how to _use_ your Azure Machine Learning assets ... ?

Reply:
Updated.